src/test/resources/features/accounts/Login.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Login/Logout
```

Comment:
Can you move this feature file into somewhere else?
I am assuming this branch is for travel scenarios.

Reply:
Done